### PR TITLE
TPD-892 - Fixed Bundle creation redirect issue and added unit test

### PR DESCRIPTION
**Pull request by jrred**
Previously when creating a new Bundle (Administration > Bundles > click "Create New Bundle" button > fill out fields > click "Save"), the page would just redirect back to itself, instead of advancing to the confirmation page.  This was due to the querystring in the redirect somehow resetting the "create" parameter to "true", even if explicitly set to false in the redirect statement executed right after the Bundle is saved.  So I added a "sentinel" using TempData to maintain the state of "Bundle Created" to prevent the faulty redirect.
touchpointsoftware.atlassian.net/browse/TPD-892

### CODE REVIEW

File: CmsWeb/Areas/Finance/Controllers/BundleController.cs [hunk 1]
```diff
@@ -22,7 +22,12 @@ public BundleController(IRequestManager requestManager) : base(requestManager)
         public ActionResult Index(int id, bool? create, bool? edit)
         {
             var m = new Models.BundleModel(id, CurrentDatabase);
-            ViewBag.createbundle = create;
+
+            if (TempData["BundleCreated"] == null)
```
Comment (brondavies) [inline CmsWeb/Areas/Finance/Controllers/BundleController.cs:26]: @jrred  How does this fix the problem?  We don't reference TempData anywhere (anymore).

**Comment by jrred** [comment]
@brondavies Fixed a couple of additional navigation problems. Link to edit an existing bundle and canceling out of edit mode were not working. Got those fixed.

File: CmsWeb/Web.config [hunk 2]
```diff
@@ -45,7 +45,7 @@
     <!-- use (local) or .\SQLEXPRESS -->
     <add key="dbserver" value="" />
     <!--Second half of db name like CMS_bellevue-->
-    <add key="host" value="" />
+    <add key="host" value="myBvcms" />
```
Comment (brondavies) [inline CmsWeb/Web.config:48]: You'll have to undo this

File: CmsWeb/Web.config [hunk 3]
```diff
@@ -1033,7 +1033,7 @@
   <system.net>
     <mailSettings>
       <smtp deliveryMethod="SpecifiedPickupDirectory">
-        <specifiedPickupDirectory pickupDirectoryLocation="c:\email" />
+        <specifiedPickupDirectory pickupDirectoryLocation="c:\dev\pursuant\email" />
```
Comment (brondavies) [inline CmsWeb/Web.config:1036]: this too

**Comment by jrred** [comment]
web.config changes reverted.

**Pull request closed without merging**